editoast/src/views/infra/objects.rs

```diff
@@ -155,6 +158,56 @@ mod tests {
         assert_eq!(call_service(&app, req).await.status(), StatusCode::OK);
     }
 
+    #[rstest]
+    async fn get_objects_should_return_switch() {
+        // GIVEN
+        let app = create_test_service().await;
+        let empty_infra = empty_infra(db_pool()).await;
+        let empty_infra_id = empty_infra.id();
+
+        let switch = Switch {
+            id: "switch_001".into(),
+            switch_type: "switch_type_001".into(),
+            ..Default::default()
+        }
+        .into();
+
+        let create_operation = Operation::Create(Box::new(switch));
```

Comment:
minor: keeping a stricter type and convert only when needed
```suggestion
        let create_operation = Operation::Create(Box::new(switch.into()));
```